ui/raidboss/autoplay_helper.ts

```diff
@@ -1,16 +1,29 @@
 export default class AutoplayHelper {
+  private static context: AudioContext | null = null;
```

Comment:
```suggestion
  private static context?: AudioContext;
```

Just preferring `undefined` over null here.